### PR TITLE
September recordings updated for monthly branch

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by praries880** [comment]
@MikhailTryakhov  As with 

You will need to take care of the following :
* Need to fix the failure ion the dns tests : https://azuresdkci.westus2.cloudapp.azure.com/job/powershell/7377/artifact/src/Publish/TestResults/FailingTests/Microsoft.Azure.Commands.Dns.Test.dll.html. Seems the tests need to be re-recorded
* Update the network pkg for keyvault at :
  * https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/KeyVault/Commands.KeyVault.Test/packages.config#L11
  * https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/KeyVault/Commands.KeyVault.Test/Commands.KeyVault.Test.csproj#L64
and
  * https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/KeyVault/Commands.KeyVault.Test/Commands.KeyVault.Test.Netcore.csproj#L30

File: src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.Netcore.csproj [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.Compute" Version="21.0.0" />
     <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="2.4.1" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.3.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.0-preview" />
```
Comment (praries880) [inline src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.Netcore.csproj]: fix this version, change it to 19.4.0

File: src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj [hunk 2]
```diff
@@ -46,8 +46,8 @@
     <Reference Include="Microsoft.Azure.Management.Compute">
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Compute.21.0.0\lib\net452\Microsoft.Azure.Management.Compute.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.3.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=20.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.20.0.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
```
Comment (praries880) [inline src/ResourceManager/Compute/Commands.Compute.Test/Commands.Compute.Test.csproj]: fix this version, change it to 19.4.0

File: src/ResourceManager/Compute/Commands.Compute.Test/packages.config [hunk 3]
```diff
@@ -7,7 +7,7 @@
   <package id="Microsoft.Azure.Graph.RBAC" version="3.4.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Compute" version="21.0.0" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.Network" version="19.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="20.0.0-preview" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Compute/Commands.Compute.Test/packages.config]: fix this version, change it to 19.4.0

File: src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.Netcore.csproj [hunk 4]
```diff
@@ -25,7 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Dns" Version="2.2.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.3.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.0-preview" />
```
Comment (praries880) [inline src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.Netcore.csproj]: fix this version, change it to 19.4.0

File: src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.csproj [hunk 5]
```diff
@@ -49,8 +49,8 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Dns.2.2.0-preview\lib\net452\Microsoft.Azure.Management.Dns.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.3.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=20.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.20.0.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
```
Comment (praries880) [inline src/ResourceManager/Dns/Commands.Dns.Test/Commands.Dns.Test.csproj]: fix this version, change it to 19.4.0

File: src/ResourceManager/Dns/Commands.Dns.Test/packages.config [hunk 6]
```diff
@@ -6,7 +6,7 @@
   <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Dns" version="2.2.0-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.Network" version="19.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="20.0.0-preview" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Dns/Commands.Dns.Test/packages.config]: fix this version, change it to 19.4.0

File: src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 7]
```diff
@@ -43,8 +43,11 @@
     <RunCodeAnalysis>false</RunCodeAnalysis>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.3.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+    <Reference Include="Hyak.Common, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj]: revert all these changes and just update the network SDK version here...

File: src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 8]
```diff
@@ -94,6 +114,7 @@
     <Compile Include="ScenarioTests\NetworkWatcherAPITests.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj]: why is this required?

File: src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj [hunk 9]
```diff
@@ -633,6 +654,8 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <WCFMetadata Include="Connected Services\" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj]: why is this required?

File: src/ResourceManager/Network/Commands.Network.Test/packages.config [hunk 10]
```diff
@@ -1,27 +1,29 @@
...
   <package id="Microsoft.Azure.Test.Framework" version="1.0.6179.26854-prerelease" targetFramework="net45" />
-  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.6.8" targetFramework="net45" />
+  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.9.0" targetFramework="net452" />
   <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
   <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.5.2" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.4.0" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network.Test/packages.config]: revert this

File: src/ResourceManager/Network/Commands.Network.Test/packages.config [hunk 11]
```diff
@@ -1,27 +1,29 @@
...
+  <package id="Microsoft.Rest.ClientRuntime.Azure.Authentication" version="2.4.0" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network.Test/packages.config]: revert this

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 12]
```diff
@@ -50,18 +50,28 @@
...
-      <HintPath>..\..\..\packages\Microsoft.Azure.KeyVault.Core.1.0.0\lib\net40\Microsoft.Azure.KeyVault.Core.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="Microsoft.Azure.KeyVault.Core, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.KeyVault.Core.3.0.0\lib\net452\Microsoft.Azure.KeyVault.Core.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.3.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.4.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: remove this

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 13]
```diff
@@ -50,18 +50,28 @@
...
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.2.3.13\lib\net452\Microsoft.Rest.ClientRuntime.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime.Azure, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: remove this

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 14]
```diff
@@ -50,18 +50,28 @@
...
+      <HintPath>..\..\..\packages\Microsoft.Rest.ClientRuntime.Azure.3.3.15\lib\net452\Microsoft.Rest.ClientRuntime.Azure.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: remove this

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 15]
```diff
@@ -50,18 +50,28 @@
...
+      <HintPath>..\..\..\packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="System.Runtime" />
+    <Reference Include="System.ValueTuple, Version=4.0.4.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: remove this

File: src/ResourceManager/Network/Commands.Network/Commands.Network.csproj [hunk 16]
```diff
@@ -619,6 +629,7 @@
     <Folder Include="Microsoft\Azure\Commands\Network\Models\" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/Commands.Network.csproj]: remove this

File: src/ResourceManager/Network/Commands.Network/packages.config [hunk 17]
```diff
@@ -1,6 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="AutoMapper" version="6.0.2" targetFramework="net452" />
-  <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Network" version="19.3.0-preview" targetFramework="net452" />
+  <package id="AutoMapper" version="7.0.1" targetFramework="net452" />
+  <package id="Microsoft.Azure.KeyVault.Core" version="3.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="19.4.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.13" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/packages.config]: remove this

File: src/ResourceManager/Network/Commands.Network/packages.config [hunk 18]
```diff
@@ -1,6 +1,10 @@
...
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.15" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/packages.config]: remove this

File: src/ResourceManager/Network/Commands.Network/packages.config [hunk 19]
```diff
@@ -1,6 +1,10 @@
...
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/packages.config]: remove this

File: src/ResourceManager/Network/Commands.Network/packages.config [hunk 20]
```diff
@@ -1,6 +1,10 @@
...
+  <package id="System.ValueTuple" version="4.6.0-preview1-26829-04" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Network/Commands.Network/packages.config]: remove this

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/Commands.RecoveryServices.Backup.Test.Netcore.csproj [hunk 21]
```diff
@@ -25,7 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Compute" Version="18.0.0" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.3.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.0-preview" />
```
Comment (praries880) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/Commands.RecoveryServices.Backup.Test.Netcore.csproj]: 19.4.0

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/Commands.RecoveryServices.Backup.Test.csproj [hunk 22]
```diff
@@ -35,8 +35,8 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Compute.21.0.0\lib\net452\Microsoft.Azure.Management.Compute.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.3.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=20.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.20.0.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
```
Comment (praries880) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/Commands.RecoveryServices.Backup.Test.csproj]: 19.4.0

File: src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/packages.config [hunk 23]
```diff
@@ -4,7 +4,7 @@
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Compute" version="21.0.0" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.Network" version="19.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="20.0.0-preview" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/packages.config]: 19.4.0

File: src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.Netcore.csproj [hunk 24]
```diff
@@ -25,7 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.18.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.3.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.0-preview" />
```
Comment (praries880) [inline src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.Netcore.csproj]: 19.4.0

File: src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.csproj [hunk 25]
```diff
@@ -48,8 +48,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Graph.RBAC.3.2.0-preview\lib\net45\Microsoft.Azure.Graph.RBAC.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Network, Version=19.0.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.3.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=20.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.20.0.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
```
Comment (praries880) [inline src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.csproj]: 19.4.0

File: src/ResourceManager/Sql/Commands.Sql.Test/packages.config [hunk 26]
```diff
@@ -6,7 +6,7 @@
   <package id="Microsoft.Azure.Gallery" version="2.6.2-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Graph.RBAC" version="3.2.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Network" version="19.3.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="20.0.0-preview" targetFramework="net452" />
```
Comment (praries880) [inline src/ResourceManager/Sql/Commands.Sql.Test/packages.config]: 19.4.0

**Comment by MikhailTryakhov** [comment]
@praries880  looks like changes are fixed...

**Comment by praries880** [comment]
on demand : https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/1091/
